src/packagedcode/phpcomposer.py

```diff
@@ -55,13 +55,13 @@
 
 
 class PHPComposerPackage(models.Package):
-    metafiles = ('package.json')
+    metafiles = ('composer.json')
     filetypes = ('.json',)
-    mimetypes = ('application/x-tar',)
+    mimetypes = ('text/xml',)
```

Comment:
I didn't find a case that composer.json is archived in an archive file. So I believe it the mimetype should be directory, do you agree? We could use " ('application/x-directory', 'text/directory', 'inode/directory')" instead, OK?